src/lib/RuleResultRecombinator.ts

```diff
@@ -24,135 +22,144 @@ export class RuleResultRecombinator {
 		}
 	}
 
-	private static constructCsv([pmdResults]: [string]): string {
-		// TODO: Eventually, we'll need logic to combine disparate result sets together into a single CSV, but for now we
-		//  can proceed with just PMD's results.
-		return this.pmdToCsv(pmdResults);
-	}
-
-	private static constructJunit([pmdResults]: [string]): string {
-		return this.pmdToJunit(pmdResults);
+	private static constructCsv(results: RuleResult[]): string {
+		return this.toCsv(results);
```

Comment:
If this is just going to be a one-line method that passes its one argument into another method, why do we have it? Could we not just move the contents of `this.toCsv` into this method?

Reply:
Yep.  Ditto for constructJunit